provisioning/service/tests/Config/BulkEnrollmentOperationErrorTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Test
+{
+    [TestClass]
+    public class BulkEnrollmentOperationErrorTests
+    {
+        private const string REGISTRATION_ID = "valid-registration-id";
```

Comment:
Same naming comments as in the previous PR.

Reply:
Done